index.js

```diff
@@ -610,7 +650,10 @@ class Grid extends react.Component {
  */
 async function getExtensionRepos(page = 1) {
     // www is needed or it will block with "cross-origin" error.
-    let url = `https://api.github.com/search/repositories?q=${encodeURIComponent("topic:spicetify-extensions")}&per_page=${ITEMS_PER_REQUEST}`;
+    let url = `https://api.github.com/search/repositories?q=${encodeURIComponent(`topic:spicetify-extensions`)}&per_page=${ITEMS_PER_REQUEST}`;
+    if (searchQuery?.trim() !== "") {
+        url = `https://api.github.com/search/repositories?q=${encodeURIComponent(`${searchQuery} topic:spicetify-extensions`)}&per_page=${ITEMS_PER_REQUEST}`;
```

Comment:
This would work as well, right?
```suggestion
        url = `https://api.github.com/search/repositories?q=${encodeURIComponent(`${searchQuery}+topic:spicetify-extensions`)}&per_page=${ITEMS_PER_REQUEST}`;

```

Reply:
It does yeah, gonna apply suggestion.